source/process.rst

```diff
@@ -49,20 +49,20 @@ they pass.
 If not, write them – or you'll need to resort to testing manually.
 
 
-.. index:: Python 2.5
+.. index:: Python 2.6
 
-Drop Python 2.5 and Lower
+Drop Python 2.6 and Lower
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Python 2.6 and 2.7 were released in lockstep with the early 3.x versions,
-and contain several features that make supporting both 2 and 3
+Python 2.7 was released in lockstep with earlier 3.x versions,
+and contains several features that make supporting both 2 and 3
 possible in the same codebase.
 
-Python 2.5 has been unmaintained for several years now, so any *new* code
+Python 2.6 has been unmaintained for several years now, so any *new* code
 written for it does not have much of a future.
 Bring this up with the software's maintainers.
 
-If compatibility with Python 2.5 is *really* necessary, we recommend that
+If compatibility with Python 2.6 is *really* necessary, we recommend that
 you fork the codebase, i.e. work on a copy and regularly merge in any
```

Comment:
This recommendation makes not sense now.

Reply:
So we wait on RHEL 6.7 (ends December 31, 2018)?

Reply:
No. We never drop Python 2.6 from the guide.

It would be nice to mention here that *also* dropping 2.6 might make the transition easier. The cases where supporting 2.6 is an additional burden are already pointed out later in the text.